### PR TITLE
#59: Used click for deployment cli

### LINKED ISSUE #59: Use Click for the deployment cli script of SME
Preparing the CLI interface that deploys the scripts required for SME installation with Click for
- better user experience
- simplify the implementation

### PR DESCRIPTION
**Pull request by umitbuyuksahin**
Fixes #59

### CODE REVIEW

File: exasol_sagemaker_extension/deployment/deploy_cli.py [hunk 1]
```diff
@@ -1,40 +1,37 @@
...
     import DeployCreateStatements
 
 
-def main(args):
-    logging.basicConfig(
-        format='%(asctime)s - %(module)s  - %(message)s',
-        level=logging.DEBUG)
+@click.command()
+@click.option('--host', type=str, required=True, help="db host address")
+@click.option('--port', type=str, required=True, help="db host port")
```
Comment (tomuben) [inline exasol_sagemaker_extension/deployment/deploy_cli.py:9]:
```suggestion
@click.option('--port', type=int, required=True, help="db host port")
```

Comment (tomuben) [reply exasol_sagemaker_extension/deployment/deploy_cli.py:9]: Also you could add the definition the default port of the db (8563)

File: exasol_sagemaker_extension/deployment/deploy_cli.py [hunk 2]
```diff
@@ -1,40 +1,37 @@
...
+@click.option('--user', type=str, required=True, help="db user name")
+@click.option('--pass', 'pass_', required=True, help="db user password")
```
Comment (tomuben) [inline exasol_sagemaker_extension/deployment/deploy_cli.py:11]: As pass is a Python keyword maybe it's better to rename, maybe `pwd`

File: exasol_sagemaker_extension/deployment/deploy_cli.py [hunk 3]
```diff
@@ -1,40 +1,37 @@
...
+@click.option('--schema', type=str, required=True, help="schema name")
+@click.option('--print', 'print_', type=bool, required=False,
```
Comment (tomuben) [inline exasol_sagemaker_extension/deployment/deploy_cli.py:13]: Here the same, maybe better rename it, maybe `verbose`

**Review by tomuben — COMMENTED** [review]
Just some minor changes

**Review by tkilias — APPROVED** [review]

**Review by tkilias — APPROVED** [review]

**Pull request merged**